Assets/Scripts/UI/PointerFullscreen.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.EventSystems;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PointerFullscreen : MonoBehaviour, IPointerDownHandler {
+    public Sprite OffSprite;
```

Comment:
Is the icon actually purple? Or is that an artifact of the Github interface?

If it's purple, this is out of style with other buttons. Please greyscale.